rust/crd/src/lib.rs

```diff
@@ -967,7 +967,7 @@ mod tests {
           name: simple-trino
         spec:
           image:
-            productVersion: "455"
+            productVersion: "469"
```

Comment:
and ff

Reply:
Done in dbf9409e08c5b82aff7d4724af1fa5b014e96fbf